### PR TITLE
RIA-9154 - Update PROGRESS_MIGRATED_CASE to send a Notification

### PR DESCRIPTION
**Pull request by fergusoshea**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RIA-9154

### Change description ###
Update PROGRESS_MIGRATED_CASE to send a Notification

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by ikirsanov — APPROVED** [review]

**Review by MiguelMartinRojas — APPROVED** [review]

File: src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
       "caseData": {
         "template": "minimal-appeal-started.json",
         "replacements": {
-          "isAdmin": "Yes",
```
Comment (MiguelMartinRojas) [inline src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json:12]: Why IsAdmin should be deleted? for https://github.com/hmcts/ia-case-notifications-api/pull/1286/files you are setting isAdmin to Yes

**Review by MiguelMartinRojas — CHANGES_REQUESTED** [review]

File: src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json (hunk 1 above)
Comment (fergusoshea) [reply src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json:12]: Hi Miguel,
This functional test was for the event and was breaking after adding the notification.
The options were to add more case details, so the notification steps were satisfied, or delete the isAdmin part.
I went with deleting as they are not used in the event part of the code.
Thanks

Comment (MiguelMartinRojas) [reply src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json:12]: I would say to go with satisfying the notification adding the missing fields, the reason is that isAriaMigrated case will always been isAdmin = "Yes" as well

Comment (fergusoshea) [reply src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json:12]: Update with listCaseHearingCentre

**Review by MiguelMartinRojas — APPROVED** [review]

**Pull request merged**